src/resmom/requests.c

```diff
@@ -5029,6 +5029,18 @@ req_del_hookfile(struct batch_request *preq) /* ptr to the decoded request   */
 			preq->rq_ind.rq_hookfile.rq_filename);
 		strcat(p, HOOK_FILE_SUFFIX);
 		if ((phook = find_hook(hook_name)) != NULL) {
+#ifndef WIN32
+			if (phook->event & HOOK_EVENT_EXECJOB_END) {
+				/** 
+				 * This event runs hook in the background,
+				 * and it's deferred task created while
+				 * running the hook, is required for graceful
+				 * exit of the job. 
+				 */
+				reply_ack(preq);
+				return;
+			}
+#endif
```

Comment:
We just removed all the win32 code.  Please remove this.

Reply:
Hi @bhroam this is mom code and we still have a mom in windows - we have only removed WIN32 code from server, sched and comm